File: 10818/10818.py
n = int(input())
num = list(input().split(' '))
min = 1000000
max = -1000000
for i in range(n):
    if int(num[i]) < min:
        min = int(num[i])
    if int(num[i]) > max:
        max = int(num[i])
print(str(min)+' '+str(max))

File: 10818/10818.c
#include <stdio.h>
int main() {
	int n, max=-1000000, min=1000000, num[1000000];
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		scanf("%d ", num + i);
		if (num[i] > max)
			max = num[i];
		if (num[i] < min) {
			min = num[i];
		}
	}
	printf("%d %d", min, max);
	return 0;
}